test.sh

```diff
@@ -22,7 +41,7 @@ run() {
 PASS=0
 FAIL=0
 
-if [ "$1" = "-g" ]; then
+if [ "${1-}" = "-g" ]; then
```

Comment:
Another bugfix - if you gave the script no arguments it vomits. I mean not really a big deal but meh,